soda_machine.rb

```diff
@@ -1,18 +1,40 @@
+require_relative "soda"
+
 class SodaMachine
   attr_reader :sodas, :cash
 
+  class Soda < SodaMachine
+  end
+
+
   def initialize(args = {})
     @sodas = args[:sodas]
     @cash = args[:cash]
   end
 
   def current_inventory_count
+  	@soda.count
   end
 
   def find_soda(soda_brand)
+  	if soda_brand == @soda[brand]
```

Comment:
`@soda` isn't a hash, is it? if so, where is it being set?  and there's no variable `brand` here, where is that coming from?